riff-raff.yaml

```diff
@@ -6,6 +6,8 @@ deployments:
     type: autoscaling
     dependencies:
       - cloudformation
+      - cloudformation-pillar-db
```

Comment:
We are going to deploy all 3 CFN stacks concurrently so it shouldn't slow down deployments too much